server/app/services/mongo_pubsub.rb

```diff
@@ -93,7 +101,7 @@ def unsubscribe(subscription)
   def publish(channel, data)
     self.collection.insert_one(
       channel: channel,
-      data: data,
+      data: BSON::Binary.new(Marshal::dump(data)),
```

Comment:
msgpack seems to be quite a bit faster than `Marshal`: https://gist.github.com/aishfenton/480059

So I'll go and change this to use it.